include/deal.II/base/data_out_base.h

```diff
@@ -1153,6 +1153,17 @@ namespace DataOutBase
      * or as a high-order Lagrange cell.
      *
      * Default is <tt>false</tt>.
+     *
+     * @note The ability to write data that corresponds to higher order
+     *   polynomials than simply linear or bilinear is a feature that was
+     *   only introduced in VTK sometime in 2017 or 2018.
+     *   You will need at least Paraview version 5.5 (released in the spring
+     *   or 2018) or a similarly recent version of Visit for this feature
```

Comment:
**of** 2018 (?)

Reply:
> similarly recent version of Visit

Do you have any idea what version number this would be? I think that would be useful info to include, since I would have no idea what version corresponds to what release date. We could be cautious and just mention the latest release version (as of today).

Reply:
I tried VisIt 2.13.2 (May 2018) and it does not seem to support this.